Source/Forms/Gen4/Eggs4.cpp

```diff
@@ -326,9 +331,32 @@ void Eggs4::seedToTime()
     time->raise();
 }
 
+void Eggs4::calcPoketch() {
+    QModelIndex index = ui->tableViewGenerator->currentIndex();
+    int advances = generatorModel->data(generatorModel->index(index.row(), 0)).toInt();
+
+    auto *poketch = new Poketch(advances);
+    poketch->show();
+    poketch->raise();
```

Comment:
Having a dedicated widget for this is probably overkill. This is just a fancy QMessageBox

Reply:
Did this for design purposes, easier to layout with the multiple outputs and the extra note depending on the advances. Easily up for debate to switch but want to discuss before switching it over

Reply:
Whichever you think is best. It just seemed like a fancy QMessageBox to me